site/app/templates/forum/CreatePost.twig

```diff
@@ -93,7 +93,11 @@
         {% endif %}
 
         <span class="last-edit">
-			<strong class="post_user_id">{{ visible_username }}</strong>
+            {% if post_user_info.is_OP == true and first == false %}
+			    <strong class="post_user_id" title="Original Poster">{{ visible_username }} <span style="color: #42A7FF;"><strong>OP</strong></span></strong>
```

Comment:
We shouldn't be using hardcoded colors like this.  Please use a CSS variable from `colors.css`.